providers/keap.go

```diff
@@ -27,6 +36,26 @@ func init() {
 			Subscribe: false,
 			Write:     false,
 		},
+		Modules: &ModuleInfo{
+			ModuleKeapV1: {
+				BaseURL:     "https://api.infusionsoft.com/v1",
+				DisplayName: "Version 1",
```

Comment:
Please update all the module displaynames to be self sufficient i.e. Keap version 1 or Hubspot CRM